qmcpy/stopping_criterion/_cub_qmc_ld_g.py

```diff
@@ -5,6 +5,9 @@
 from numpy import *
 from time import time
 import warnings
+import matplotlib.pyplot as plt
```

Comment:
move this to within the method performing the plotting. This makes it so matplotlib is only imported if the user wants to plot something, but does not make it a requirement of QMCPy.

Reply:
Since I plan on making an if-statement that handles plotting, I'll move that there as I make those changes.